domain/model/mesh_results.py

```diff
@@ -1,29 +1,37 @@
 from dataclasses import dataclass
 from typing import List
 
+from domain.types import AgentID
+
 
 @dataclass
 class Metric:
-    health: str
-    value: int
+    """ Represents single from->to connection metric """
+
+    health: str  # "healthy", "warning", ...
+    value: int  # latency in microseconds, jitter in microseconds, packet_loss in percents (0-100)
 
 
 @dataclass
 class MeshColumn:
-    name: str
-    alias: str
-    id: str
+    """ Represents connection "to" endpoint """
+
+    name: str  # agent name
```

Comment:
I think it is better to name it "agent_name" instead of adding comment, since to comment is visible only in one place. The same with agent alias and fields in MeshRow class.

Reply:
Note that the `name` field returned in `Agent{List,Get}` data is actually internal and should not be shown externally. It will likely be removed soon. There is a ticket open with engineering to include more metadata for agents in the API, so that they can be more uniquely identified with human consumable strings. I propose to use combination of `alias`, `id` and `asn` as "name" presented to users for now, like for example so:
```
alias (id: <id> asn: <asn>)
```

Reply:
Done